compiler/resolution/functionResolution.cpp

```diff
@@ -3514,24 +3534,29 @@ void resolveNormalCall(CallExpr* call) {
   if (call->partialTag && (!best || !best->fn ||
                            !best->fn->hasFlag(FLAG_NO_PARENS))) {
     if (best != NULL) {
-      delete best;
+      // best is deleted below with the other candidates
       best = NULL;
```

Comment:
best is also stored in the candidates array, so it will be deleted in line 3572, as far as I can tell.
As far as I can tell, it was a double-free error before, but I don't understand how to trigger it.